src/config.js

```diff
@@ -21,7 +21,7 @@
 import browser from 'bowser';
 
 const PREFIX =
-  browser.webkit ? '-webkit-' :
+  browser.webkit || browser.blink ? '-webkit-' :
```

Comment:
blink is the fork of webkit used in newer Chrome. The webkit flag is off for Chrome >= 28. https://github.com/ded/bowser#rendering-engine-flags